### ISSUE TITLE
对CUE文件的支持

### ISSUE THREAD

**Issue by GoogleCodeExporter** [memeco/osd-lyrics#147]
```
What's the feature you like?
用audacious打开一个cue文件，osd-lyrics只会对第一首歌自动搜索�
��载歌词，后面的就不行了，还是显示第一首歌的歌词，需要
手动搜索下载。

Is there any other lyric show implemented this feature?
有其他的歌词秀有这个功能吗？
应该是有的吧。
```

Original issue reported on code.google.com by `onionliu...@gmail.com` on 7 Nov 2010 at 10:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `tigerso...@gmail.com` on 28 Mar 2011 at 11:42
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
是啊,我也遇到这个了,听cue的时候只显示第一首歌的歌词
```

Original comment by `lands...@gmail.com` on 9 Sep 2011 at 11:32

**Comment by GoogleCodeExporter** [comment]
```
when will this bug be solved? 
I use the newest version 0.4.1, still have this bug......
Need I do some config ?
```

Original comment by `magicsto...@gmail.com` on 29 Nov 2011 at 3:54

**Comment by GoogleCodeExporter** [comment]
```
Maybe the lyrics should mapping to which one listed on the playlist but not the 
file on the disk.
```

Original comment by `magicsto...@gmail.com` on 29 Nov 2011 at 3:57

**Comment by GoogleCodeExporter** [comment]
```
A quick hack to this bug.
Works with 0.4.1 & 0.4.2b1.

```

Original comment by `wanderhuang` on 9 Dec 2011 at 1:14

Attachments:
- [liclib-4.x.patch](https://storage.googleapis.com/google-code-attachments/osd-lyrics/issue-147/comment-5/liclib-4.x.patch)

**Comment by GoogleCodeExporter** [comment]
```
bug fixed

Will be released with 0.4.2
```

Original comment by `tigerso...@gmail.com` on 20 Dec 2011 at 3:10
- Changed state: **Fixed**
